### PR TITLE
Added `responsive_is_wpdoc` to parent theme.

### PR DESCRIPTION
**Pull request by hirozed**
This is the first part.  The child themes that use `is_wpdoc` have been updated to use the new function.

Fixes https://github.com/bu-ist/responsive-framework-1x/issues/421

### Changes proposed in this pull request
* Removed individual `is_wpdocs()` from the following themes:
  * https://github.com/bu-ist/r-socialwork
    * https://github.com/bu-ist/r-socialwork/pull/8
  * https://github.com/bu-ist/r-sargent
    * https://github.com/bu-ist/r-sargent/pull/66
  * https://github.com/bu-ist/r-precisiondiag
    * https://github.com/bu-ist/r-precisiondiag/pull/24
  * https://github.com/bu-ist/r-studenthealth
    * https://github.com/bu-ist/r-studenthealth/pull/37
* Created `responsive_is_wpdocs()` as a substitute
* Added to 1x responsi: https://github.com/bu-ist/responsive-framework-1x/pull/424

Deploy-wise, the responsi themes should go out first, then the child themes.

### Review checklist

[x] I've reviewed the [contribution guidelines](https://github.com/bu-ist/coding-standards/blob/develop/CONTRIBUTING.md).
[x] I've updated `CHANGELOG.MD` with a brief explanation of the main changes in this pull request.
[x] My code follows [BU Coding Standards](https://github.com/bu-ist/coding-standards).

### CODE REVIEW

File: inc/extras.php [hunk 1]
```diff
@@ -344,3 +344,12 @@ function responsive_get_layout_default() {
 
 	return $new_default;
 }
+
+function responsive_is_wpdocs() {
+	$subdomain = explode( '.', $_SERVER['HTTP_HOST'] )[0];
+	if ( $subdomain && 'wpdocs' == $subdomain ) {
```
Comment (stonybrookweb) [inline inc/extras.php]: Please use strict comparisons `===`

**Comment by ashleykolodziej** [comment]
Hey @hirozed! I was just taking a quick peek at this and I think you may have accidentally bumped the version on `style.css` instead of `css-dev/style.scss` :) can you bump that instead and run `grunt build`?

**Comment by hirozed** [comment]
@ashleykolodziej Of course.

**Comment by hirozed** [comment]
@ashleykolodziej Updated here and in the 1x version as well

**Pull request merged**